Nickvision.Aura/Keyring/SystemCredentialManager.cs

```diff
@@ -1,47 +1,41 @@
+using DBus.Services.Secrets;
 using Meziantou.Framework.Win32;
 using System;
+using System.Collections.Generic;
 using System.Runtime.InteropServices;
```

Comment:
Can this be removed?

Reply:
InteropServices? No, we need `RuntimeInformation.IsOSPlatform`